data_processors/pipeline/domain/workflow.py

```diff
@@ -106,29 +106,29 @@ def get_workdir_root(self):
     def get_output_root(self):
         return libssm.get_ssm_param(self.get_ssm_key_output_root())
 
-    def construct_workdir(self, target_id, timestamp: datetime):
+    def construct_workdir(self, target_id, timestamp: datetime, secondary_target_id:str = None, portal_run_uid: str=None):
```

Comment:
Can we please use `portal_run_id` consistently throughout code base; whenever we meant/expect for that value, instead of `portal_run_uid`.